### DISCUSSION TITLE
Meeting #3: Date TBD

### DISCUSSION THREAD

**Discussion by AngelOnFira** [rust-gamedev/wg#118] [Meetings]
Opening this for people to add agenda suggestions for the meeting next week.

The date will be selected once I hear back from the other WG team members about what times work best for them.

**Comment by ozkriff** [comment]
- Sharing access to WG's domains (arewegameyet.rs, arewegameyet.com, arewegameyet.org, gamedev.rs)

**Comment by Wodann** [comment]
- What is the desired output from/goal for WG meetings? If we're unsure, how can we determine the desired output/goal?
